ambry-store/src/main/java/com.github.ambry.store/CompactionLog.java

```diff
@@ -313,13 +327,13 @@ static CycleLog fromBytes(DataInputStream stream, StoreKeyFactory storeKeyFactor
      */
     Phase getPhase() {
       Phase phase;
-      if (copyStartTime == -1) {
+      if (copyStartTime == UNINITIALIZED_TIMESTAMP) {
         phase = Phase.PREPARE;
-      } else if (switchStartTime == -1) {
+      } else if (commitStartTime == UNINITIALIZED_TIMESTAMP) {
         phase = Phase.COPY;
-      } else if (cleanupStartTime == -1) {
-        phase = Phase.SWITCH;
-      } else if (cycleEndTime == -1) {
+      } else if (cleanupStartTime == UNINITIALIZED_TIMESTAMP) {
+        phase = Phase.COMMIT;
+      } else if (cycleEndTime == UNINITIALIZED_TIMESTAMP) {
         phase = Phase.CLEANUP;
       } else {
         phase = Phase.DONE;
```

Comment:
when are these times reset? When does it go from all set to all uninitialized? (just trying to understand the transition from DONE to PREPARE)